src/c-unchop.c

```diff
@@ -85,7 +85,7 @@ r_obj* list_unchop(r_obj* xs,
     out_size += r_length(r_list_get(indices, i));
   }
 
-  r_obj* locs = KEEP(vec_as_indices(indices, out_size, r_null));
+  r_obj* locs = KEEP(list_as_locations(indices, out_size, r_null));
```

Comment:
`vec_as_indices()` took a list of location vectors and called `vec_as_location()` on each. I've renamed that to `list_as_locations()` to reflect modern naming schemes. It might not be a perfect name but I think it is much closer.